resources/3rdparty/glpk-5.0/Makefile.am

```diff
@@ -2,6 +2,6 @@
 
 ACLOCAL_AMFLAGS=-I m4
 
-SUBDIRS = src examples
```

Comment:
We probably should add glpk to the `update_resources.md` and document what you have done to get rid of the examples folder.

Reply:
Good point, thanks for the comment.
I added the documentation.